examples/earthquake-dashboard/app.py

```diff
@@ -0,0 +1,74 @@
+import json
+from pathlib import Path
+
+import pandas as pd
+import panel as pn
+
+from panel_gwalker import GraphicWalker
+
+ROOT = Path(__file__).parent
+CSS = ROOT / "app.css"
+DATASET = ROOT / "significant_earthquake_dataset_1900_2023.parquet"
+SPEC = ROOT / "spec.json"
+
+@pn.cache
+def get_css():
+    return CSS.read_text()
+
+
+@pn.cache
+def get_df() -> pd.DataFrame:
+    df = pd.read_parquet(DATASET)
+    df["Time"] = pd.to_datetime(df["Time"]).dt.strftime("%Y-%m-%d %H:%M:%S")
+    df.to_parquet("significant_earthquake_dataset_1900_2023.parquet", index=False)
+    return df
+
+
+@pn.cache
+def get_spec():
```

Comment:
here it would be very nice if the `spec` parameter would take (file) paths as input.